src/Microsoft.ComponentDetection.Contracts/TypedComponent/DotNetComponent.cs

```diff
@@ -0,0 +1,64 @@
+namespace Microsoft.ComponentDetection.Contracts.TypedComponent;
+
+using System.Text;
+
+#nullable enable
```

Comment:
is this line required?

Reply:
I wanted to use `nullable` for the new code it adds an additional level of static validation.